assets/js/app/LocusZoom.js

```diff
@@ -3,25 +3,40 @@
 /* eslint-disable no-console */
 
 var LocusZoom = {
-    version: "0.3.6"
+    version: "0.3.7"
 };
     
 // Populate a single element with a LocusZoom instance.
 // selector can be a string for a DOM Query or a d3 selector.
-LocusZoom.populate = function(selector, datasource, layout, state) {
+LocusZoom.populate = function(selector, datasource, layout) {
     if (typeof selector == "undefined"){
         throw ("LocusZoom.populate selector not defined");
     }
+    // Empty the selector of any existing content
+    d3.select(selector).html("");
+    // If state was passed as a fourth argument then merge it with layout (for backward compatibility)
+    if (typeof state != "undefined"){
```

Comment:
My mistake - I had removed it from the arguments list on `populate` and `populateAll` originally on this branch but set out to restore it per our recent conversation. I added the processing of a state argument but just forgot to add it back to the arguments list.